### ISSUE TITLE
About Mental Health

### ISSUE THREAD

**Issue by jhembe** [CodeSaphire/mHealth#7]
# About section
contains introductory bits of information concerning Mental Health, providing all necessary knowledge for the page visitors to get an idea of what they're getting themselves into. Make this page as presentable as possible, adhere to the colour coding, themes and the design language we agreed on

**Comment by bilalshaikh292** [comment]
Heyy is the issue still open ? I would Like to Contribute.

**Comment by jhembe** [comment]
Yes it's open, so please contribute. Work with it on the testing branch then you can merge it when it's done

**Comment by bilalshaikh292** [comment]
Ok i will do it asap
